erpnext_com/www/pricing/faq.md

```diff
@@ -0,0 +1,67 @@
+<section class='top-section'>
+	<div class='container'>
+		<h1>Pricing FAQs</h1>
+        <p class="lead">Some common questions related to our services<p>
+   </div>     
+</section>
+
+##### Q. How is the fees calculated for the number of users?
+You only pay for the number of System users in your accounts. i.e those who can access the desk.
+
+##### Q. Who is a system user?
```

Comment:
lets explain this without using "Desk". We can say, system user is someone who uses core modules - not portals. Customers, Suppliers, Students, Guardians etc are not system users.

Desk is going to change in v12 (pretty soon)